service.autosubs/resources/settings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Would be nice to start working on the new settings format:

https://forum.kodi.tv/showthread.php?tid=350135

Reply:
will do on next commit (light default.py)